dev/lint-python

```diff
@@ -38,7 +38,7 @@ function compile_python_test {
 
     # compileall: https://docs.python.org/2/library/compileall.html
```

Comment:
Should we change this comment? Looks like it points to Python 2?

Reply:
Sure. Thanks!